frontend/src/app/main/ui/workspace/shapes.cljs

```diff
@@ -75,7 +77,15 @@
    ::mf/wrap-props false}
   [props]
   (let [shape (obj/get props "shape")
-        opts  #js {:shape shape}]
+
+        active-frames
+        (when (cph/root-frame? shape) (mf/use-ctx ctx/active-frames-ctx))
+
+        thumbnail?
+        (and (some? active-frames)
+             (not (contains? active-frames (:id shape))))
+
+        opts  #js {:shape shape :thumbnail? thumbnail?}]
     (when (and (some? shape) (not (:hidden shape)))
       [:*
```

Comment:
The `[:*` is not necessary here I think.

Reply:
OK